src/hotspot/share/opto/vectornode.cpp

```diff
@@ -260,9 +260,9 @@ int VectorNode::opcode(int sopc, BasicType bt) {
   case Op_SignumD:
     return Op_SignumVD;
   case Op_ConvHF2F:
-    return Op_HF2FV;
+    return Op_VectorCastH2F;
```

Comment:
Could we use the same name style with the scalar op, i.e. `"Op_VectorCastHF2F"` ?

Reply:
I support this suggestion. Lets be consistent.